app/models/manageiq/providers/openstack/cloud_manager/template.rb

```diff
@@ -73,6 +73,29 @@ def requires_storage_for_scan?
     false
   end
 
+  def self.raw_create_image(ext_management_system, create_options)
+    ext_management_system.with_provider_connection(:service => 'Compute') do |service|
+      service.images.create(create_options)
```

Comment:
Nit: I think we usually use request methods (instead of fog model's one), search for ```image_create``` at https://github.com/fog/fog-openstack/tree/master/lib/fog/compute/openstack/requests